src/integration-test/helpers/clients/idamClient.ts

```diff
@@ -31,6 +37,43 @@ export class IdamClient {
     })
   }
 
+  /**
+   * Authenticate user
+   *
+   * @param {string} email
+   * @param password the users password (optional, default will be used if none provided)
+   * @returns {Promise<string>}
+   */
+  static async authenticateUser (username: string, password: string = undefined): Promise<string> {
+
+    password = password || defaultPassword
+
+    const base64Authorisation: string = IdamClient.toBase64(`${username}:${password}`)
+    const oauth2Params: string = IdamClient.toUrlParams(oauth2)
+
+    const authResponse = await request.post({
+      url: `${baseURL}/oauth2/authorize?response_type=code&${oauth2Params}`,
```

Comment:
There could be also one private `authorize` method that does most of the heavy lifting  which is called by `authenticateUser`, `authenticatePinUser` and `upliftUser` method as all these methods needs OAuth2 query parameters.